### PR TITLE
[SHELL32] Use 1996 as birth year in IDD_ABOUT for consistency

### PR DESCRIPTION
**Pull request by JoachimHenze**
We do use the birth year 1996 instead of 1998
within our msgina pictures already,
and also within the [SDK] for autogenerated file-info.

Since SHELL32 IDD_ABOUT is used by many applications it makes sense to harmonize with that.

Beside that, strip some redundancy from the headers.

## Side-note to @binarymaster
Please do not suggest further changes within the header sections, I don't want to transform all their styles
yet. If you want to do that, you can do it yourself later in your own PR.
I stripped no information, I just made that a bit more compact, keeping the existing info.

### CODE REVIEW

File: dll/win32/shell32/lang/eu-ES.rc [hunk 1]
```diff
@@ -1,7 +1,6 @@
 /*
  * PROJECT:     ReactOS Shell32
  * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
- * PURPOSE:     Spanish (Basque) resource file
```
Comment (binarymaster) [inline dll/win32/shell32/lang/eu-ES.rc:4]: Incorrect deletion actually.

File: dll/win32/shell32/lang/zh-TW.rc [hunk 2]
```diff
@@ -1,7 +1,6 @@
 /*
  * PROJECT:     ReactOS Shell32
  * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
- * PURPOSE:     Chinese (Traditional) resource file
```
Comment (HBelusca) [inline dll/win32/shell32/lang/zh-TW.rc:4]: I think you can keep those lines.

**Review by HBelusca — APPROVED** [review]

File: dll/win32/shell32/lang/eu-ES.rc (hunk 1 above)
Comment (JoachimHenze) [reply dll/win32/shell32/lang/eu-ES.rc:4]: No, look at line 7.

File: dll/win32/shell32/lang/zh-TW.rc (hunk 2 above)
Comment (JoachimHenze) [reply dll/win32/shell32/lang/zh-TW.rc:4]: 100% redundancy. Look at line 9.

File: dll/win32/shell32/lang/eu-ES.rc (hunk 1 above)
Comment (HBelusca) [reply dll/win32/shell32/lang/eu-ES.rc:4]: Well, it says Basque language.

Comment (binarymaster) [reply dll/win32/shell32/lang/eu-ES.rc:4]: Line 7 is for automated tools, it doesn't contain the description of a file.

Comment (JoachimHenze) [reply dll/win32/shell32/lang/eu-ES.rc:4]: LANGUAGE LANG_BASQUE, SUBLANG_NEUTRAL
contains everything one needs to know, and avoids unnecessary typos like ukraiainian, ukrianian, ukraiaiaian. ukrain.
I asked to please not discuss the headers. You will work over all headers anyway in a later PR.
Either propose anything for further shortening, i would accept such a thing.
Or keep it in your own PR please.

**Review by binarymaster — CHANGES_REQUESTED** [review]
PR title states "Use 1996 as birth year in IDD_ABOUT for consistency", however you're attempting to sneak in a change for translation headers (removal of "PURPOSE" field) which is not correct. Your argument is "redundancy", but I could say the same for code files which purpose is obvious to me -- "Nice, let's remove that!" - This is how your thinking works.

**Comment by JoachimHenze** [comment]
Show me a single .rc file in our whole source tree of more than 12000 files where an actual information in within the purpose field! You will not find a single one!

But you know what, have it your way: In 5years when 99.9% of our PRs will be only about grooming our redundant stuff and stupid typos in our headers, you will hopefully understand me.

**Comment by binarymaster** [comment]
> I asked to please not discuss the headers.

Yes, you asked it to sneak these changes in, so you can later use it for justification.

> You will work over all headers anyway in a later PR.

Some headers here don't need a rework at all, if you're going to merge these removals, I will have to just revert some of them. I have enough things to work on, don't make me spend additional time on these.

**Comment by binarymaster** [comment]
> Show me a single .rc file in our whole source tree of more than 12000 files where an actual information is within the purpose field! You will not find a single one!

There is such thing as Standard. Our coding style defines it.

**Comment by JoachimHenze** [comment]
Hvae your fun with the headers yourself then for the next 30 years.

**Pull request closed without merging**